cpp/src/arrow/builder-benchmark.cc

```diff
@@ -383,51 +364,25 @@ static void BM_ArrayDataConstructDestruct(
 
 // ----------------------------------------------------------------------
 // Benchmark declarations
+//
+
+BENCHMARK(ReferenceBuildVectorNoNulls);
```

Comment:
You're not enclosing this is in an `#ifdef ARROW_WITH_BENCHMARKS_REFERENCE`. Is it an oversight?

Reply:
yes.